azure-pipelines.yml

```diff
@@ -787,26 +787,34 @@ jobs:
       name: 'linux-pool'
       demands: assignment -equals default
     variables:
-      pr.num: $[ variables['System.PullRequest.PullRequestNumber'] ]
       branch_sha: $[ dependencies.git_sha.outputs['out.branch'] ]
       status: $[ dependencies.collect_build_data.result ]
       is_release: $[ dependencies.check_for_release.outputs['out.is_release'] ]
     steps:
+      - checkout: self
+        persistCredentials: true
+      - template: ci/bash-lib.yml
+        parameters:
+          var_name: bash-lib
       - bash: |
           set -euo pipefail
 
-          tell_daml() {
-              local message pr_handler
-              message=$1
-              pr_handler=$(head -1 release/rotation | awk '{print $1}')
-              curl -XPOST \
-                   -i \
-                   -H 'Content-Type: application/json' \
-                   --data "{\"text\":\"<@${pr_handler}> <https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|Build $(Build.BuildId)> for release PR <https://github.com/digital-asset/daml/pull/$(pr.num)|#$(pr.num)> has completed with status ${message}.\"}" \
-                   $(Slack.team-daml)
-          }
-
-          tell_daml "$(status)"
+          source $(bash-lib)
+
+          AUTH="$(get-gh-auth-header)"
+          PR=$(curl -H "$AUTH" \
+                    -H "Accept: application/vnd.github.groot-preview+json" \
+                    -s -f \
+                    "https://api.github.com/repos/digital-asset/daml/commits/$(git rev-parse HEAD)/pulls" \
+               | jq '.[0].number' \
+               || echo "")
```

Comment:
do you care about the newline? If not `true` seems like a more natural choice here.

Reply:
I don't. I keep forgetting about `true` for some reason. Given that it's already merged I don't think this is worth changing, but I'll try to keep it in mind next time.

Reply:
Also note that the newline is actually discarded by the assignment here:
```
bash-3.2$ a=$(fail || echo "")
bash: fail: command not found
bash-3.2$ b=$(fail || true)
bash: fail: command not found
bash-3.2$ declare -p a
declare -- a=""
bash-3.2$ declare -p b
declare -- b=""
bash-3.2$ 
```